client/layout/store-alerts/style.scss

```diff
@@ -89,7 +89,7 @@
 	}
 }
 
-.woocommerce-store-alerts__message {
+.components-card__body .woocommerce-store-alerts__message {
```

Comment:
In the new version, we need to add specificity to override `<Card>` component's default margin